setup.cfg

```diff
@@ -20,4 +20,6 @@ packages = find:
 zip_safe = False
 python_requires = >=3.11
 install_requires =
-    requests>=2.31.0
\ No newline at end of file
+    requests>=2.31.0
+    requests-mock>=1.11.0
```

Comment:
I think this is in the wrong place, but it does work. I'll likely need to provide a test environment yaml for the CI down the track.